patches/api/0163-Make-the-default-permission-message-configurable.patch

```diff
@@ -43,15 +43,20 @@ index ca4e2d3b27f629e0d5e672fc915a5d03f0c0581d..17f8dd9870a47227a7c9bb09cceedb94
       * Creates a PlayerProfile for the specified uuid, with name as null
       * @param uuid UUID to create profile for
 diff --git a/src/main/java/org/bukkit/command/Command.java b/src/main/java/org/bukkit/command/Command.java
-index 7c80dc54776d0d66f7816b77136f6dbd9b801704..c10fc8d2386301bc2caddcdb1cd18566bcaa8689 100644
+index 7c80dc54776d0d66f7816b77136f6dbd9b801704..811eda94a354928e3b1824e9636a37bee0052a9e 100644
 --- a/src/main/java/org/bukkit/command/Command.java
 +++ b/src/main/java/org/bukkit/command/Command.java
-@@ -185,7 +185,7 @@ public abstract class Command {
+@@ -185,7 +185,12 @@ public abstract class Command {
          }
  
          if (permissionMessage == null) {
 -            target.sendMessage(ChatColor.RED + "I'm sorry, but you do not have permission to perform this command. Please contact the server administrators if you believe that this is a mistake.");
-+            target.sendMessage(Bukkit.getPermissionMessage()); // Paper
++            // Paper start
++            String bukkitPermissionMessage = Bukkit.getPermissionMessage();
++            if (!bukkitPermissionMessage.equals("")) {
```

Comment:
`isEmpty()` might be better? maybe also call `trim()` to catch a string which is just ` `

Reply:
StringUtils has a better method for this as it can handle null too, not that we should be expecting that

Reply:
I was ignoring null as everything involved was `@NonNull`, but sure I can handle that too.